src/ProjectModule/components/ProjectsData/ProjectsData.tsx

```diff
@@ -27,122 +35,138 @@ interface FormData{
   const getProjects = async () => {
     try {
       const response = await axios.get(
-        `https://upskilling-egypt.com:3003/api/v1/Project/${param.id?param.id : ""}`,
+        `https://upskilling-egypt.com:3003/api/v1/Project/${
+          param.id ? param.id : ""
+        }`,
         {
           headers: { Authorization: `Bearer ${Token}` },
         }
       );
-    //   console.log(rsponse.data.data);
-    setValue("title", response.data.title);
-    setValue("description", response.data.description);
-
+      //   console.log(rsponse.data.data);
+      setValue("title", response.data.title);
+      setValue("description", response.data.description);
     } catch (error) {
-      console.log(error)
+      console.log(error);
     }
   };
 
- 
-  const addProject: SubmitHandler<FormData> =async(data)=>{
-   try {
-   const addProgect= await axios.post(`https://upskilling-egypt.com:3003/api/v1/Project`,data,
-    {headers:{Authorization: `Bearer ${Token}`}}
-    )
-    console.log(addProgect);
-    goToProject();
-    
-   } catch (error) {
-    console.log(error)
-   }
-
-  }
+  const addProject: SubmitHandler<FormData> = async (data) => {
+    try {
+      const addProgect = await axios.post(
+        `https://upskilling-egypt.com:3003/api/v1/Project`,
+        data,
+        { headers: { Authorization: `Bearer ${Token}` } }
+      );
+      console.log(addProgect);
+      goToProject();
+    } catch (error) {
+      console.log(error);
```

Comment:
You need to give the user a descriptive feedback. logging the error to the console is for you as a developer in the developing process, but the user needs to get a suitable feedback